app.json

```diff
@@ -1,18 +1,13 @@
 {
-  "name": "thorin-cfh",
+  "name": "cfh",
   "scripts": {
   },
-  "env": {
-    "MONGOHQ_URL": "mongodb://thorin:thorin@ds147659.mlab.com:47659/thorin-cfh",
```

Comment:
Is this the url to the live database? Do you consider it safe to have this here?